spec/unit/blockscore/candidate_spec.rb

```diff
@@ -2,19 +2,45 @@ module BlockScore
   RSpec.describe Candidate do
     let(:api_stub) { @api_stub }
 
-    it 'updates' do
-      candidate = create(:candidate_params)
-      candidate.name_first = 'Chris'
-      expect(candidate.save).to eq(true)
-      assert_requested(api_stub, times: 2)
-      expect('Chris').to eq(candidate.name_first)
+    let(:candidate) { create(:candidate_params) }
+
+    context 'updating' do
+      before do
+        candidate.name_first = 'Chris'
+        candidate.save
+      end
+
+      let(:uri)         { "/candidates/#{candidate.id}"                     }
+      let(:http_method) { :patch                                            }
+      let(:expected)    { { body: hash_including('name_first' => 'Chris') } }
+
+      it { expect(candidate.name_first).to eql('Chris') }
+      include_context 'api request'
+    end
+
+    describe '#delete' do
+      before { candidate.delete }
+
+      let(:uri)         { "/candidates/#{candidate.id}" }
+      let(:http_method) { :delete                       }
+
+      include_context 'api request'
     end
 
-    it 'deletes' do
-      candidate = create(:candidate_params)
-      candidate.delete
-      assert_requested(api_stub, times: 2)
-      expect(true).to eq(candidate.deleted)
+    describe '#search' do
+      let(:constraints) { { name_first: 'John' } }
+      subject(:search) { -> { candidate.search(constraints) } }
+
+      context 'search request' do
+        let(:uri)      { "/watchlists"                                         }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.